AGENT_SYSTEM_PROMPT.md

```
@@ -0,0 +1,761 @@
+**`get_message_recording`** / **`get_message_transcription`** / **`download_transcription`** — Access call data
+- Params: `messageId`*
+- Example trigger: *"Get the transcript for call message msg456"*
+
+**`add_inbound_message`** — Log an inbound message manually
+- Params: `contactId`*, `source`, `body`
+
+**`add_outbound_call`** — Log an outbound call
+- Params: `contactId`*, `duration`
+
+**`cancel_scheduled_message`** / **`cancel_scheduled_email`**
+- Params: `scheduledMessageId`* / `scheduledEmailId`*
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)